frontend/views/containers/user-settings/Troubleshooting.vue

```diff
@@ -104,41 +74,15 @@ export default ({
         }
       })
     },
-    dummy3secTask () {
-      return new Promise((resolve, reject) => {
-        setTimeout(resolve, 3000)
-      })
-    },
     async startResync () {
       if (this.ephemeral.status === 'ok' && !confirm(L('Are you sure you want to re-sync your app data? This might take a few minutes.'))) {
         return null
       }
 
-      this.ephemeral.status = 'recovering'
-      this.$refs.doneMsg.clean()
-
       try {
-        // Dummy logic, obviously.
-        this.updateProgress(L('Deleting local data...'), 0.25)
-        await this.dummy3secTask()
-        this.updateProgress(L('Downloading new data...'), 0.50)
-        await this.dummy3secTask()
-        this.updateProgress(L('Last touches...'), 0.90)
-
-        // Change false to true to force a dummy error
-        if (false) { // eslint-disable-line
-          throw Error('Dummy error forced')
-        }
-
-        await this.dummy3secTask()
-
-        // All done!
+        this.ephemeral.status = 'recovering'
```

Comment:
Per the issue, we need:

> When they click it, let them know they'll be logged out and ask them to confirm if they're sure.

In other words the text above needs to be changed to something like:

```
This will reset Group Income and log you out. You will need to log back in with your password.

Are you sure?
```